sklearn/pipeline.py

```diff
@@ -272,6 +278,22 @@ def inverse_transform(self, X):
         return Xt
 
     @if_delegate_has_method(delegate='_final_estimator')
+    def pipe(self, X, y):
+        """Applies transformations to X and y. Valid only if the final
+        estimator implements pipe.
+
+        Parameters
+        ----------
+        X : iterable
+            Data to inverse transform. Must fulfill output requirements of the
```

Comment:
Damn, the docstrings are not finished yet, it seems. I'll fix them now. Also add some narrative.